content/quizzes/en.yaml

```
@@ -529,7 +529,7 @@ levels:
                 -   "option_text": "{at} {random} now needs quotation marks"
                     "feedback": "No, but 2 other commands do."
             correct_answer: "A"
-            hint: "In level 4 you need quotation marks for 2 commands."
+            hint: "In level 4 you need quotation marks {for} 2 commands."
```

```suggestion
            hint: "In level 4 you need quotation marks for 2 commands."
```

This for is not a command :)